### PR TITLE
QA report for 0.37.x alpha3

### PR DESCRIPTION
**Pull request by lasarojc**
[QA docs README rendered](https://github.com/cometbft/cometbft/blob/lasarojc/qa_0.37.x-alpha3/docs/qa/README.md)

---

#### PR checklist

- [x] Tests written/updated
- [ ] Changelog entry added in `.changelog` (we use [unclog](https://github.com/informalsystems/unclog) to manage our changelog)
- [x] Updated relevant documentation (`docs/` or `spec/`) and code comments

### CODE REVIEW

File: docs/qa/v037/CometBFT.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+The changes with respect to the baseline, `TM v0.37.x` as of Oct 12, 2022 (Commit: 1cf9d8e276afe8595cba960b51cd056514965fd1), include the rebranding of our fork of Tendermint Core to CometBFT and several improvements, described in the CometBFT [CHANGELOG](https://github.com/cometbft/cometbft/blob/v0.37.0-alpha.3/CHANGELOG.md).
+
+## Testbed
+
+As in other iterations of our QA process, we have used a 200-node network as testbed, plus nodes to introduce load and collect metrics.
+
+### Saturation point
+
+As previous iterations, in our QA experiments, the system is subjected to a load slightly under a saturation point.
```
Comment (thanethomson) [inline docs/qa/v037/CometBFT.md:21]:
```suggestion
As in previous iterations, in our QA experiments, the system is subjected to a load slightly under a saturation point.
```

File: docs/qa/v037/CometBFT.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+### Peers
+
+The number of peers was stable at all nodes.
+It was higher for the seed nodes (around 140) than for the rest (between 16 and 78).
+The red dashed line denotes the average value.
+
+![peers](./img/200nodes_cmt037/peers.png)
+
+Just as in the baseline, shown next, the fact that non-seed nodes reach more than 50 peers is due to #9548.
```
Comment (thanethomson) [inline docs/qa/v037/CometBFT.md:78]:
```suggestion
Just as in the baseline, shown next, the fact that non-seed nodes reach more than 50 peers is due to [tendermint/tendermint\#9548](https://github.com/tendermint/tendermint/issues/9548).
```

Do we have a corresponding issue in the CometBFT repo? The original one was automatically closed by stalebot.

File: docs/qa/v037/CometBFT.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+
+
+
+[\#9533]: https://github.com/tendermint/tendermint/pull/9533
+[\#9534]: https://github.com/tendermint/tendermint/pull/9534
+[\#9539]: https://github.com/tendermint/tendermint/issues/9539
+[\#9548]: https://github.com/tendermint/tendermint/issues/9548
+[\#9537]: https://github.com/tendermint/tendermint/issues/9537
+[\#9581]: https://github.com/tendermint/tendermint/issues/9581
```
Comment (thanethomson) [inline docs/qa/v037/CometBFT.md:172]: Are all of these links referenced in the preceding Markdown text?

**Review by thanethomson — APPROVED** [review]

File: docs/qa/v037/CometBFT.md (hunk 3 above)
Comment (lasarojc) [reply docs/qa/v037/CometBFT.md]: not really. removed.

File: docs/qa/v037/CometBFT.md (hunk 2 above)
Comment (lasarojc) [reply docs/qa/v037/CometBFT.md]: We don't.
Using the references at the end.

File: docs/qa/v037/CometBFT.md (hunk 1 above)
Comment (lasarojc) [reply docs/qa/v037/CometBFT.md]: fixed.

**Review by sergio-mena — COMMENTED** [review]
This looks really good. I'll try to run the updated scripts on the baseline data (as we did for `v0.34.x` so the comparison is even clearer.

**Pull request merged**